src/goToLocation.ts

```diff
@@ -1,13 +1,17 @@
 import { workspace, window, ViewColumn, Uri, Range } from "vscode";
 import { DocumentUri, Range as LspRange } from "vscode-languageclient/node";
+import { MetalsFileProvider } from "./metalsContentProvider";
 
 export interface WindowLocation {
   uri: DocumentUri;
   range: LspRange;
   otherWindow: boolean;
 }
 
-export function gotoLocation(location: WindowLocation): void {
+export function gotoLocation(
+  location: WindowLocation,
+  metalsFileProvider: MetalsFileProvider | undefined = undefined
```

Comment:
```suggestion
  metalsFileProvider?: MetalsFileProvider
```
should have the same effect